theme/colors.js

```diff
@@ -0,0 +1,28 @@
+export const COLORS = {
+  // brand
+  primary: '#2d40cc',
+  secondary: '#90efe4',
+  tertiary: '#008dc5',
+
+  // priority
+  success: '#a3ef90',
+  info: this.primary,
+  warning: '#ffc107',
+  danger: '#a72b1e',
+
+  // utility
+  paper: {
+    main: '#fff',
+    inverted: this.text.main
+  },
+  text: {
+    main: '#1a1a1a',
+    inverted: this.paper.main
+  },
+  grey: {
+    extralight: '#d8d8d8',
+    light: '#4d4d4d',
+    medium: '#666',
+    dark: '#464a4c',
+  },
+};
```

Comment:
not sure how this is handled in the other projects, but maybe add a newline at the end of this line (same for the other two files)?

Reply:
our linter automatically saves a blank last line, I can look into our linter configs and see what can be reused here.

Reply:
i can add the rule `eol-last` to it, that will make sure any file ends with a newline (think that is what we used in our other linting as well)